core/lib/zksync_core/src/state_keeper/updates/l1_batch_updates.rs

```diff
@@ -56,9 +56,9 @@ mod tests {
     use zksync_types::{MiniblockNumber, ProtocolVersionId, H256};
 
     use super::*;
-    use crate::{
-        gas_tracker::new_block_gas_count,
-        state_keeper::tests::{create_execution_result, create_transaction},
+    use crate::state_keeper::{
+        tests::{create_execution_result, create_transaction},
+        utils::new_block_gas_count,
```

Comment:
Nit: Looks like this import and similar imports in other test modules are redundant because of the `use super::*` wildcard import above.

Reply:
`super::*` give access only for `updates` mod in this case..so I think we need these imports from `crate::state_keeper::{`